Web/Edubase.Web.UI/Areas/Groups/Views/Group/SearchCompaniesHouse.cshtml

```diff
@@ -31,7 +31,15 @@
         <div class="govuk-grid-column-full">
             @Html.Partial("_ValidationSummary", ViewData.ModelState)
             <h1 id="create-academy-trust-details-heading" class="govuk-heading-xl">Create new academy trusts</h1>
```

Comment:
You haven't changed the heading as is required by the ticket

Reply:
Sorry think I just rushed this one and didn't read all of the story.  Changed it now.